dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/view/RenderingTemplateRepositoryImpl.java

```diff
@@ -13,29 +15,33 @@
 import org.springframework.stereotype.Repository;
 
 @Repository
-public class RenderingTemplateRepositoryImpl extends AbstractPagingAndSortingRepositoryImpl
+public class RenderingTemplateRepositoryImpl extends JdbiRepositoryImpl
     implements RenderingTemplateRepository {
 
-  public static final String MAPPING_PREFIX = "r";
+  public static final String MAPPING_PREFIX = "rt";
   public static final String TABLE_ALIAS = "rt";
   public static final String TABLE_NAME = "rendering_templates";
 
-  private final Jdbi dbi;
+  public static final String SQL_INSERT_FIELDS =
+      " uuid, created, label, description, name, last_modified";
+  public static final String SQL_INSERT_VALUES =
+      " :uuid, :created, :label::JSONB, :description::JSONB, :name, :lastModified";
+  public static final String SQL_REDUCED_FIELDS_RT =
+      String.format(
+          " %1$s.uuid, %1$s.created, %1$s.label, %1$s.name, %1$s.last_modified", TABLE_ALIAS);
+  public static final String SQL_FULL_FIELDS_RT =
+      SQL_REDUCED_FIELDS_RT + String.format(", %s.description", TABLE_ALIAS);
 
-  public RenderingTemplateRepositoryImpl(Jdbi dbi) {
-    this.dbi = dbi;
-  }
-
-  @Override
-  public long count() {
-    String sql = "SELECT count(*) FROM rendering_templates";
-    long count = dbi.withHandle(h -> h.createQuery(sql).mapTo(Long.class).findOne().get());
-    return count;
+  public RenderingTemplateRepositoryImpl(Jdbi dbi, CudamiConfig cudamiConfig) {
+    super(
+        dbi, TABLE_NAME, TABLE_ALIAS, MAPPING_PREFIX, cudamiConfig.getOffsetForAlternativePaging());
   }
 
   @Override
   public PageResponse<RenderingTemplate> find(PageRequest pageRequest) {
-    StringBuilder query = new StringBuilder("SELECT * FROM rendering_templates");
+    StringBuilder query =
+        new StringBuilder(
+            "SELECT " + SQL_REDUCED_FIELDS_RT + " FROM " + tableName + " AS " + tableAlias);
```

Comment:
```suggestion
            String.format("SELECT %2$s.* FROM %1$s AS %2$s", tableName, tableAlias));
```
Wenn `addPageRequestParams` aufgerufen wird, sollte lieber der `*` verwendet werden (nur so ist bei großen Offsets eine Optimierung möglich).